src/Symfony/Component/Intl/Tests/NumberFormatter/AbstractNumberFormatterTest.php

```diff
@@ -641,11 +646,13 @@ public function parseProvider()
         );
     }
 
-    /**
-     * @expectedException \PHPUnit_Framework_Error_Warning
-     */
     public function testParseTypeDefault()
     {
+        if (class_exists('PHPUnit_Framework_Error_Warning')) {
+            $this->setExpectedException('\PHPUnit_Framework_Error_Warning');
```

Comment:
you said this method is deprecated? should be do a method_exists check and use the new way when possible?